app/docked-window.js

```diff
@@ -100,6 +100,11 @@ class DockManager {
         var screenHeight = screenSize().height;
         var x = this._anchor.position(window._index);
         var y = screenHeight - window.height;
+		var os = require('os');
+
+		if(os.type().startsWith("Windows")){
+			y -= 40
```

Comment:
It would be better to use `os.platform() === 'win32'`, and declared up top as a constant.

Also, the style is a space after `if` and before the opening `{`, and end the line with a semicolon.